art_web_app/views.py

```diff
@@ -1,5 +1,29 @@
 from django.shortcuts import render
+from artworks.models import Artwork, ArtCategory, Category
+from django.core.exceptions import ObjectDoesNotExist
+import random
 
 
 def home_view(request):
-    return render(request, "home.html", {})
\ No newline at end of file
+    invalid_ids = set()
+    
+    try:
+        nr_of_artworks = Artwork.objects.latest('id').id
```

Comment:
It's not OK to get the number of items like this because it's not actually the real number of items. Imagine the following:
- add 3 items: ID = 1, ID = 2, ID = 3
- delete one item (either ID = 1 or ID = 2)
- what's the value your code will have for `nr_of_artworks`?
You can use `Artwork.objects.all().count()`